spec/acceptance/post_spec.rb

```diff
@@ -570,4 +570,68 @@
       end
     end
   end
+
+  describe "timezone handling" do
+    before do
+      freeze_time Time.utc(2022, 7, 24, 13, 00)
+    end
+
+    it "stores the correct information in the database" do
+      expected_datetime = ActiveSupport::TimeZone["Australia/Sydney"].parse("2022-07-24 14:01")
+
+      p = PostCreator.create!(
+        user,
+        title: 'Beach party',
+        raw: "[event start='2022-07-24 14:01' timezone='Australia/Sydney']\n[/event]"
+      ).reload
+
+      expect(p.event.timezone).to eq("Australia/Sydney")
+      expect(p.event.original_starts_at).to eq_time(expected_datetime)
+      expect(p.event.starts_at).to eq_time(expected_datetime)
+      expect(p.event.event_dates.first.starts_at).to eq_time(expected_datetime)
```

Comment:
```suggestion
      post = PostCreator.create!(
        user,
        title: 'Beach party',
        raw: "[event start='2022-07-24 14:01' timezone='Australia/Sydney']\n[/event]"
      ).reload

      expect(post.event.timezone).to eq("Australia/Sydney")
      expect(post.event.original_starts_at).to eq_time(expected_datetime)
      expect(post.event.starts_at).to eq_time(expected_datetime)
      expect(post.event.event_dates.first.starts_at).to eq_time(expected_datetime)
```

IMO, the characters we save typing here are not worth the impact on readability.